spec/controllers/users/reset_passwords_controller_spec.rb

```diff
@@ -21,16 +27,27 @@
 
         expect(@analytics).to have_received(:track_event).
           with('Password Reset: Token Submitted', analytics_hash)
+        expect(@irs_attempts_api_tracker).to have_received(:track_event).with(
+          :forgot_password_email_confirmed,
+          success: false,
+          failure_reason: irs_tracker_failure_reason,
```

Comment:
I think it would be clearer and have less indirection if we just inlined this value? (same for the rest of the file)
```suggestion
          failure_reason: { user: ['invalid_token'] },
```